pycognaize/common/utils.py

```diff
@@ -628,3 +630,8 @@ def cloud_interface_login(login_instance: Login) -> CloudInterface:
     else:
         ci_instance = CloudInterface()
     return ci_instance
+
+
+def generate_directory_summary_hash(directory: str) -> str:
```

Comment:
Do not check the directory checksum, only the folders. You should exclude the files.

Reply:
Also add a docstring please